src/generic.rs

```diff
@@ -1,13 +1,17 @@
 //! This module contains the actual, albeit generic, implementaiton of the `Cow`,
 //! and the traits that are available to it.
 
-use alloc::borrow::{Borrow, Cow as StdCow};
+use alloc::borrow::{Borrow, Cow as StdCow, ToOwned};
 use alloc::string::String;
 use alloc::vec::Vec;
 use core::cmp::Ordering;
 use core::fmt;
 use core::hash::{Hash, Hasher};
+#[cfg(feature = "const_deref")]
+use core::marker::Destruct;
 use core::marker::PhantomData;
+#[cfg(not(feature = "const_deref"))]
+use core::marker::Sized as Destruct;
```

Comment:
@CAD97 is there anything more invasive than Sized

Reply:
`where Self: Sized` is in fact a trivial bound that doesn't add any restrictions on the ungated impl, if that's what you're asking.

Reply:
Yes. I just don't know anything more trivial.

Reply:
`T: ?Sized` is the only weaker bound than `T: ?Sized` currently, or just `T:`.

Reply:
Yes, but we can't create alias for this